src/main/java/com/redhat/cloud/notifications/models/Endpoint.java

```diff
@@ -47,14 +47,15 @@
     @JsonFormat(shape = JsonFormat.Shape.STRING)
     private Date updated;
 
-    @Schema(oneOf = { WebhookAttributes.class, EmailAttributes.class })
+    @Schema(oneOf = { WebhookAttributes.class, EmailAttributes.class, DefaultAttributes.class })
     @JsonTypeInfo(
             use = JsonTypeInfo.Id.NAME,
             property = "type",
             include = JsonTypeInfo.As.EXTERNAL_PROPERTY)
     @JsonSubTypes({
             @JsonSubTypes.Type(value = WebhookAttributes.class, name = "webhook"),
-            @JsonSubTypes.Type(value = EmailAttributes.class, name = "email")
+            @JsonSubTypes.Type(value = EmailAttributes.class, name = "email"),
+            @JsonSubTypes.Type(value = DefaultAttributes.class, name = "default")
```

Comment:
Why? It's not mandatory item.

Reply:
When sending: 
```json
{"type":"default","name":"Default endpoint type","description":"","enabled":true,"properties":{}}
```
you get: 
`com.fasterxml.jackson.databind.exc.InvalidTypeIdException: Could not resolve type id 'default' as a subtype of `com.redhat.cloud.notifications.models.Attributes`: known type ids = [email, webhook] (for POJO property 'properties') at [Source: (io.quarkus.vertx.http.runtime.VertxInputStream); line: 1, column: 96]`

When sending:
```json
{"type":"default","name":"Default endpoint type","description":"","enabled":true}
```
you get:
`com.fasterxml.jackson.databind.exc.MismatchedInputException: Missing property 'properties' for external type id 'type' at [Source: (io.quarkus.vertx.http.runtime.VertxInputStream); line: 1, column: 81] (through reference chain: com.redhat.cloud.notifications.models.Endpoint["properties"])`

It looks like that is required by using it like that?

Reply:
Something else that worked is to use `io.smallrye.common.constraint.Nullable` on `properties` and send `properties: null`.

Reply:
> Something else that worked is to use `io.smallrye.common.constraint.Nullable` on `properties` and send `properties: null`.

Ah, nevermind. It looks like just sending `properties: null` works already...